src/dashboards/teachers/details.jsonnet

```
@@ -12,7 +12,7 @@ local common = import '../common.libsonnet';
 dashboard.new(
   'Course video details',
   tags=[common.tags.xapi, common.tags.video, common.tags.teacher],
-  editable=false,
+  editable=true,
```

Don't forget to switch this back before merging!

```suggestion
  editable=false,
```